### PR TITLE
kubeadm: add test coverage to cmd.go

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:
this PR adds test coverage for `cmd.go` in kubeadm.
one thing to test here is to validate the list of commands from `NewKubeadmCommand()` against a predefined list.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/cmd_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestNewKubeadmCommand(t *testing.T) {
```
Comment (dixudx) [inline cmd/kubeadm/app/cmd/cmd_test.go]: I don't think we really need this test, which has already been covered by tests in [`cobra.Command`](https://github.com/spf13/cobra/blob/master/command_test.go).

Comment (neolit123) [reply cmd/kubeadm/app/cmd/cmd_test.go]: ok, waiting on confirmation before closing this PR.

Comment (neolit123) [reply cmd/kubeadm/app/cmd/cmd_test.go]: @kad, @dmmcquay should this be closed?

Comment (kad) [reply cmd/kubeadm/app/cmd/cmd_test.go]: No strong opinion from me, but I think we can close.

**Pull request closed without merging**